src/frame_timings.rs

```diff
@@ -4,21 +4,21 @@
 
 use glib::translate::*;
 use ffi;
-use nonzero_signed::NonZeroI64;
+use std::num::NonZeroU64;
 use FrameTimings;
 
 impl FrameTimings {
     #[cfg(any(feature = "v3_8", feature = "dox"))]
-    pub fn get_predicted_presentation_time(&self) -> Option<NonZeroI64> {
-        NonZeroI64::new(unsafe {
-            ffi::gdk_frame_timings_get_predicted_presentation_time(self.to_glib_none().0)
+    pub fn get_predicted_presentation_time(&self) -> Option<NonZeroU64> {
+        NonZeroU64::new(unsafe {
+            ffi::gdk_frame_timings_get_predicted_presentation_time(self.to_glib_none().0) as u64
```

Comment:
Maybe add an assertion here for negative numbers, and below

Reply:
Of course!